backend/routes/eventAttendees.js

```diff
@@ -61,16 +64,43 @@ router.post('/event/:event_id/add/:volunteer_id', async (request, response, next
                 volunteerId
             }
             const volunteerRequest = await eventAttendeesQueries.signupVolunteerForEvent(postData);
-    
+            const event = await eventQueries.getSingleEvent(postData.eventId)
+            const admin = await userQueries.getAllAdmin();
+
+
+            sgMail.setApiKey(process.env.SENDGRID_API_KEY);
+            const msg = {
+                personalizations: [{
+                    to: []
+                }],
+                from: 'endeavorapp2020@gmail.com',
+                subject: 'Volunteer Event Request',
+                text: `${new Date().toLocaleString()}: ${request.user.v_first_name} ${request.user.v_last_name} requested to volunteer for the '${event.topic}' event.`,
+            };
+            for (let i = 0; i < admin.length; i++) {
+                msg.personalizations[0].to.push({ email: admin[i].a_email })
+            }
+            (async () => {
+                try {
```

Comment:
Maybe remove the try/catch for sending the emails
if an error occurs it will be handled by the outer try/catch sending a response to the frontend (the volunteer in this case)
This is open for discussion because maybe we want to personalize the message sent to the frontend and in this case we keep the try/catch and we throw a new error in the catch with the personalized message